api-tests/features/p1-journeys.feature

```diff
@@ -0,0 +1,248 @@
+@Build
+Feature: P2 journey
+
+  Scenario: P1 App Journey
+    Given I start a new 'low-confidence' journey with feature set 'p1Journeys'
+    Then I get a 'page-ipv-identity-document-start' page response
+    When I submit an 'appTriage' event
+    Then I get a 'dcmaw' CRI response
+    When I submit 'kenneth-driving-permit-valid' details to the CRI stub
+    Then I get a 'page-dcmaw-success' page response
+    When I submit a 'next' event
+    Then I get an 'address' CRI response
+    When I submit 'kenneth-current' details to the CRI stub
+    Then I get a 'fraud' CRI response
+    When I submit 'kenneth-score-1' details to the CRI stub
+    Then I get a 'page-ipv-success' page response
+    When I submit a 'next' event
+    Then I get an OAuth response
+    When I use the OAuth response to get my identity
+    Then I get a 'P1' identity
+    And an 'IPV_IDENTITY_ISSUED' audit event was recorded [local only]
+
+  Scenario: P1 Nino Journey
```

Comment:
Isn't this a duplicate of 'P1 No Photo Id Journey' in `p1-no-photo-id.feature`?

Reply:
Hmm, it's almost identical actually, only difference is that this one uses a 2 score instead of 1 for the KBV result. I'll delete this test